sdk/aqueduct/integrations/s3_integration.py

```diff
@@ -53,7 +55,7 @@ def file(
                 Description of the query.
```

Comment:
Can we update the docstring here? Specifically the requirement between artifact_type and format.